scripts/Interface/ui/sans_isis/add_runs_page.py

```diff
@@ -23,6 +24,7 @@ def __init__(self, parent=None):
         super(AddRunsPage, self).__init__(parent)
         self.setupUi(self)
         self._connect_signals()
+        UsageService.registerFeatureUsage("Feature", "ISIS SANS - Add Runs Tab", False)
```

Comment:
Does this report usage when the tab is used, or when the GUI is constructed? The latter seems less useful.

Reply:
Currently the presenter handles the connections for the view, this makes it quite difficult to keep our feature reporting on in the view rather than leaking into the presenter.

As add runs is pretty simplistic and not an obscure feature, i.e. will probably have medium usage relative to everything else, I'll drop this line until the future